Services/Sources/Services/LocationService/LocationService.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+import UIKit
+
+public protocol LocationServiceHolder {
+    var locationService: LocationService { get }
+}
+
+// sourcery: AutoMockable
+public protocol LocationService {
```

Comment:
Same as in https://github.com/AnimealProject/animeal_iOS/pull/20/files#r906061300